docs/content/preview/reference/configuration/yb-tserver.md

```diff
@@ -209,6 +209,96 @@ Stop attempting to log to disk if the disk is full.
+| :--- | :--- | :--- | :--- | :--- |
+| TServer %  | 45% | 48% | 57% | 60% |
+| master %   | 20% | 15% | 10% | 10% |
+| Postgres % | 25% | 27% | 28% | 27% |
+| other %    | 10% | 10% |  5% |  3% |
+
+To read this table, take your node's available memory in GiB and find the column who's heading interval it falls under.  For example, a 7 GiB node would fall in the (4,8] interval because it is bigger than 4 and less than or equal to 8.  The defaults for [`--default_memory_limit_to_ram_ratio`](#default-memory-limit-to-ram-ratio) on this node will thus be `0.48` for TServers and `0.15` for masters. The Postgres and other percentages are not set via gflag today but rather consist of whatever memory is left after TServer and master take their cut.
+
+For comparison, when `--use_memory_defaults_optimized_for_ysql` is `false`, the split is TServer 85%, master 10%, Postgres 0%, and other 5%.
+
+The defaults for the within TServer partitioning flags when `--use_memory_defaults_optimized_for_ysql` is true do not depend on the node size and are:
+
+| flag | default |
+| :--- | :--- |
+| --db_block_cache_size_percentage | 32 |
+| --tablet_overhead_size_percentage | 10 |
+
+_still being written..._
+
+
+
+
+
+### Flags controlling the split of memory among processes
+
+Note that in general these flags will have different values for TServer and master processes.
+
+##### --memory_limit_hard_bytes
+
+Maximum amount of memory this process should use in bytes, its hard memory limit.  A value of `0` specifies to instead use a percentage of the total system memory; see [`--default_memory_limit_to_ram_ratio`](#default-memory-limit-to-ram-ratio) for the percentage used.  A value of `-1` disables all memory limiting.
+
+Default: `0`
+
+##### --default_memory_limit_to_ram_ratio
+
+The percentage of available RAM to use for this process if [`--memory_limit_hard_bytes`](#memory-limit-hard-bytes) is `0`.  The special value `-1000` means to instead use the default value for this flag.  Available RAM excludes memory reserved by the kernel.
+
+Default: `0.85` unless [`--use_memory_defaults_optimized_for_ysql`](#use-memory-defaults-optimized-for-ysql) is true.
+
+
+### Flags controlling the split of memory within a TServer
+
+##### --db_block_cache_size_bytes
+
+Size of the shared RocksDB block cache (in bytes).  A value of `-1` specifies to instead use a percentage of this processes' hard memory limit; see [`--db_block_cache_size_percentage`](#db_block_cache_size_percentage) for the percentage used.  A value of `-2` disables the block cache.
+
+Default: `-1`
+
+##### --db_block_cache_size_percentage
+
+Percentage of our hard memory limit to use for the shared RocksDB block cache, if [`--db_block_cache_size_bytes`](#db-block-cache-size-bytes) is that `-1`.  The special value `-1000` means to instead use the default value for this flag.  The special value `-3` means to use a older default that does not take the amount of RAM into account.
```

Comment:
```suggestion
Percentage of Yugabyte's hard memory limit to use for the shared RocksDB block cache, if [`--db_block_cache_size_bytes`](#db-block-cache-size-bytes) is `-1`.  The special value `-1000` means to instead use the default value for this flag.  The special value `-3` means to use an older default that does not take the amount of RAM into account.
```

Reply:
changed Yugabyte's to process';  made other fixes